app/src/main/java/com/dmonzonis/nookpendium/CollectionActivity.kt

```diff
@@ -107,34 +104,36 @@ class CollectionActivity : AppCompatActivity() {
         viewAdapter.setRecords(recordset.records)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.toolbar_menu, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
     override fun onOptionsItemSelected(item: MenuItem?): Boolean {
         // Close all drawers before to avoid having multiple drawers opening on top of each other
         closeAllDrawers()
-        return if (drawerToggle.onOptionsItemSelected(item)) {
-            true
-        } else when (item?.itemId) {
+        when (item?.itemId) {
+            android.R.id.home -> {
```

Comment:
why is this android.R and the one below just R?

Reply:
because this is not a custom ID, it's the default ID given by android to the 'home' button

Reply:
ah ok, good to know